chromium_src/chrome/browser/ui/views/toolbar/toolbar_ink_drop_util.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/views/chrome_layout_provider.h"
+
+#define ConfigureInkDropForToolbar ConfigureInkDropForToolbar_UnUsed
```

Comment:
Should include `chrome/browser/ui/views/toolbar/toolbar_ink_drop_util.h` so that we don't redefine the symbol in the header?

Reply:
Didn't get any issue as this file doesn't have any caller of it. but added to not touch decls in that header.